docs/examples/quickstart/test_quickstarts.rb

```diff
@@ -9,12 +9,15 @@
 CURL_ERROR = "curl: (7) Failed to connect to localhost port 5000: Connection refused\n"
 CURL_EMPTY = "curl: (52) Empty reply from server\n"
 
+
 quickstarts = [
-  { lang: 'go', setup: 'go build', server: './oso-go-quickstart' },
-  { lang: 'java', setup: 'make install', server: 'make run' },
-  { lang: 'nodejs', setup: 'npm i', server: 'npm run dev' },
-  { lang: 'python', setup: 'pip install --upgrade -r requirements.txt', server: 'FLASK_APP=app.server python -m flask run' },
-  { lang: 'ruby', setup: 'bundle', server: 'bundle exec ruby server.rb' },
+  { lang: 'go', local_setup: "go mod edit -replace github.com/osohq/go-oso@v0.20.1=../../../../languages/go && go build", release_setup: 'go build', server: './oso-go-quickstart' },
+  # TODO: add local_setup that tests against local Oso install for java
+  { lang: 'java', local_setup: "make install", release_setup: 'make install', server: 'make run' },
+  # TODO: add local_setup that tests against local Oso install for nodejs
```

Comment:
Re: this TODO, check out the RBAC guides which use `yarn link` to do this in node: https://github.com/osohq/oso/commit/383e8e404bbcc7af51714e923897babdb75659f8#diff-567779331322c80ce60cbbdeeab9826ce8928573bacef49f5e30ec4efdb48499R8

Reply:
Yeah maybe we can do this as a follow-up--since it's not using yarn now I don't really want to mess with it and hold up the PR.